presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -309,6 +282,12 @@ public CostAssertionBuilder memoryUnknown()
             return this;
         }
 
+        public CostAssertionBuilder hasUnknownComponents()
```

Comment:
We still allow unknown components. Cost calculator might set it explicitly. I am good either way, but I will leave it as is for now since it matches `statistics-12` code.